src/Components/Articles.jsx

```diff
@@ -0,0 +1,40 @@
+import {useEffect, useState} from 'react'
+
+
+const Articles = () => {
+
+    const [articles, setArticles] = useState([])
+    
+    useEffect(() => {
+        fetch(`https://project-the-first.herokuapp.com/api/articles`)
+        .then((response) => response.json())
+        .then(({articles}) => {
+          console.log(articles, "In the Article useEffect")
+            setArticles(articles)
+            console.log(articles, "Tag")
+        })
+    },[])
+
+    
+    
+    return (
+        <div className='articlesList'>
+            <h2>All the Articles</h2>
+            {articles.map((article) => {
+                return (
+                  <ul className="articlePage">
```

Comment:
This could be extracted out into its own component here :) perhaps an "article card" 🤔